### PR TITLE
GraphQL - user media reactions + votes

### PR DESCRIPTION
**Pull request by toyhammered**
# What

Add media reactions to a user. This will also need to add media reactions to a library entry.

# Checklist
Must Merge https://github.com/hummingbird-me/kitsu-server/pull/662 first

- [ ] All files pass Rubocop
- [ ] Any complex logic is commented
- [ ] Any new systems have thorough documentation
- [ ] Any user-facing changes are behind a feature flag (or: explain why they can't be)

- [ ] All the tests pass
- [ ] Tests have been added to cover the new code

### CODE REVIEW

File: app/graphql/types/media_reaction_vote.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Types::MediaReactionVote < Types::BaseObject
```
Comment (NuckChorris) [inline app/graphql/types/media_reaction_vote.rb]: Should this match the approach on Likes, where you just have a list of users?

**Review by NuckChorris — CHANGES_REQUESTED** [review]

File: app/graphql/types/media_reaction_vote.rb (hunk 1 above)
Comment (toyhammered) [reply app/graphql/types/media_reaction_vote.rb]: yeah it should, I'll fix it up.

File: app/graphql/types/media_reaction_vote.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  field :author, Types::Profile,
+    null: false,
+    description: 'Profile who upvoted this media reaction.',
+    method: :user
+
+  field :media_reaction, Types::MediaReaction,
+    null: false,
+    description: 'Reaction that was upvoted.'
+end
```
Comment (NuckChorris) [inline app/graphql/types/media_reaction_vote.rb]: Is this type used anywhere or can it be deleted?

File: app/graphql/types/library_entry.rb [hunk 3]
```diff
@@ -15,9 +15,9 @@ class Types::LibraryEntry < Types::BaseObject
     null: false,
     description: ''
 
-  # field :media_reaction, Types::MediaReaction,
-  #   null: true,
-  #   description: 'Reaction for this specific media'
+  field :media_reaction, Types::MediaReaction,
```
Comment (NuckChorris) [inline app/graphql/types/library_entry.rb]: Thinking maybe this should just be `:reaction` to cut unnecessary context

**Review by NuckChorris — CHANGES_REQUESTED** [review]

File: app/graphql/types/media_reaction_vote.rb (hunk 2 above)
Comment (toyhammered) [reply app/graphql/types/media_reaction_vote.rb]: good catch totally forgot to delete.

File: app/graphql/types/library_entry.rb (hunk 3 above)
Comment (toyhammered) [reply app/graphql/types/library_entry.rb]: yes.

**Review by NuckChorris — APPROVED** [review]

**Pull request merged**